lib/wit/gitrepo.py

```diff
@@ -89,6 +95,17 @@ def clone(self, source, name):
             else:
                 raise
 
+    # Only newer git versions can use '--reference-if-able', so we emulate the 'if-able'.
+    def git_reference_capable(self):
+        env_ref_path = os.getenv("WIT_WORKSPACE_REFERENCE")
+        paths = [Path(env_ref_path) / self.name,
+                 Path(env_ref_path) / (self.name+'.git')]
+        for path in paths:
+            log.warn("{} {}".format(path, os.path.isdir(path)))
```

Comment:
This isn't really a warning, right, since there's nothing unusual or unexpected about this codepath? I think you want this to be DEBUG, SPAM, or TRACE (I always forget the levels that we've defined here).

Reply:
It's not a warning, should have removed entirely